core/settings_base.py

```diff
@@ -182,6 +183,21 @@
 
 CRISPY_TEMPLATE_PACK = "tailwind"
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'loggers': {
+        '': {
+            'handlers': ['console'],
+            'level': 'INFO',
+        },
+    },
+}
```

Comment:
It's better to move the `LOGGING` configuration to the appropriate environment specific settings rather than adding it to the base settings.